lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
When I commented previously I had a look at the composer docs and couldn't see a way of achieving this without namespaces.

Hence the question and request:

> Is there away to remove the need for the require_once statements without namespacing? If so, a link to a reference would be super-helpful.

The power of Google brings me to http://codingexplained.com/coding/php/composer-autoloading-third-party-libraries-without-namespaces

I'll take a look and update accordingly.